src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,240 @@
+sidebar: true
+lang: en
+---
+
+# Ethereum open areas of research
+
+One of the primary strengths of Ethereum is that it is constantly being improved by an active research and engineering community. There are many enthusiastic, skilled people worldwide that would like to apply themselves to outstanding issues in Ethereum, but it is not always easy to find out what those issues are. This page aims to outline some of the key areas of active research as a rough guide to Ethereum's cutting edge.
+
+## General research resources
+
+Regardless of the specific topic, there is a wealth of information on Ethereum research to be found at [ethresear.ch](https://ethresear.ch) and the [Eth R&D Discord channel](https://discord.gg/n7JxAeRu). These are the primary places where Ethereum researchers discuss the latest ideas and development opportunities.
+
+## Scaling and Performance
+
+There are ongoing efforts to scale Ethereum using a variety of techniques including sharding the blockchain, danksharding and using rollups. Introductory information on scaling Ethereum is available on our [scaling page](https://ethereum.org/en/developers/docs/scaling).
+
+### Layer 2
+
+There are now several Layer 2 protocols that scale Ethereum using different techniques for batching transactions and securing them on Ethereum layer 1. This is a very rapidly growing topic with a lot of research and development potential.
+
+#### Links
+
+[Our Layer-2 page](https://ethereum.org/en/layer-2/)
+
+[EthHub Layer-2](https://docs.ethhub.io/ethereum-roadmap/layer-2-scaling/zk-rollups/)
+
+[ethresear.ch Layer 2](https://ethresear.ch/c/layer-2/32)
+
+
+### Bridges
+
+One particular area of Layer-2 that requires more research and development is safe and performant bridges. This includes bridges between various Layer-2s and bridges between Layer 1 and Layer 2. This is a particularly important area of researchg because bridges are commonly targeted by hackers.
+
+#### Links
+
+[Our Bridges page](https://ethereum.org/en/developers/docs/bridges)
+
+[Vitalik on bridges](https://old.reddit.com/r/ethereum/comments/rwojtk/ama_we_are_the_efs_research_team_pt_7_07_january/hrngyk8/)
+
+
+### Sharding
+
+Sharding Ethereum's blockchain has long been part of the development roadmap. However, new scaling solutions such as "Danksharding" are currently taking center stage. 
+
+#### Links
+
+[Our Sharding page](https://ethereum.org/en/develoeprs/docs/shard-chains)
+
+[Proto-Danksharding notes](https://notes.ethereum.org/@vbuterin/proto_danksharding_faq)
+
+[Bankless Danksharding video](https://www.youtube.com/watch?v=N5p0TB77flM)
+
+### Hardware
+
+Running nodes (see our [nodes page](https://ethereum.org/en/developers/docs/nodes-and-clients/run-a-node)) on modest hardware is fundamental to keeping Ethereum decentralized. Therefore, active research into minimizing the hardware requirements to run nodes is an important area of research.
```

Comment:
```suggestion
[Running nodes](/developers/docs/nodes-and-clients/run-a-node/) on modest hardware is fundamental to keeping Ethereum decentralized. Therefore, active research into minimizing the hardware requirements to run nodes is an important area of research.
```
General (opinionated) note on links - feel free to just link directly in the content using relevant anchor text. No need to say "our xyz page", people know that a link takes them to a new page. We keep a consistent formatting so all internal looks have one style, while all external links have an arrow (↗) afterwards to signify it takes them to a different site.